packages/bootstrap/scss/_variables.scss

```diff
@@ -1392,6 +1392,65 @@ $spreadsheet-drawing-outline-width: 2px !default;
 $spreadsheet-drawing-anchor-bg: rgba( $header-selected-bg, .25 ) !default;
 
 
+// Timeline
+
+$timeline-padding-x: 40px !default;
+$timeline-padding-y: 40px !default;
+$timeline-items-padding: $padding-x !default;
+
+$timeline-mobile-padding-x: $padding-x !default;
+$timeline-mobile-padding-y: $padding-x !default;
+
+$timeline-track-arrow-width: 38px !default;
+$timeline-track-arrow-height: 38px !default;
+
+$timeline-track-arrow-disabled-text: #7a7c7e !default;
+$timeline-track-arrow-disabled-bg: #eff1f3 !default;
+$timeline-track-arrow-disabled-border: #eff1f3 !default;
+
+$timeline-track-size: 6px !default;
+$timeline-track-border-width: 1px !default;
+$timeline-track-margin-bottom: $spacer !default;
+$timeline-track-start-calc: calc(#{$timeline-track-arrow-width} - 2 * #{$timeline-track-border-width}) !default;
+$timeline-track-end-calc: calc(#{$timeline-track-arrow-width} - 2 * #{$timeline-track-border-width}) !default;
+$timeline-track-bg: $button-bg !default;
+$timeline-track-border-color: $timeline-track-bg !default;
+
+$timeline-track-event-min-width: 90px !default;
+$timeline-track-event-offset: 36px !default;
+
+$timeline-vertical-padding: 80px !default;
+$timeline-vertical-padding-with-dates-calc: calc(#{$timeline-track-event-min-width} + #{$timeline-padding-x}) !default;
```

Comment:
I do not understand what this var name means. Could you rename it to a more meaningful one?